tests/exporters/exporters_utils.py

```diff
@@ -67,6 +67,7 @@
     "data2vec-audio": "hf-internal-testing/tiny-random-Data2VecAudioModel",
     "deberta": "hf-internal-testing/tiny-random-DebertaModel",
     "deberta-v2": "hf-internal-testing/tiny-random-DebertaV2Model",
+    "decision-transformer": "edbeeching/decision-transformer-gym-hopper-medium",
```

Comment:
is there a smaller variant ?

Reply:
There are 3 variants - expert, medium, medium-replay

- 'edbeeching/decision-transformer-gym-hopper-medium'
- 'edbeeching/decision-transformer-gym-hopper-medium-replay'
- 'edbeeching/decision-transformer-gym-hopper-expert'

All are of the same size - 6.6 MB

Reply:
oh okay that's small enough 👍